public/controllers/management/components/management/groups/groups-table.js

```diff
@@ -78,7 +78,16 @@ class WzGroupsTable extends Component {
   }
 
   async componentDidUpdate(prevProps, prevState) {
-    if (this.props.state.isProcessing && this._isMounted) {
+    /**
+      Is verifying that isProcessing is true and that it has changed its value,
+      since in the shouldComponentUpdate it is making it re-execute several times
+      each time a state changes, regardless of whether it is a change in isProcessing.
+     */
+    if (
+      prevProps.state.isProcessing !== this.props.state.isProcessing &&
+      this.props.state.isProcessing &&
+      this._isMounted
+    ) {
```

Comment:
**nitpick**: taking in account that this section is overcomplicated, and the method to get the items is repeated 2 times. I guess, ensuring one request is done could reduce the case where 2 requests could be triggered by the `componentDidUpdate` method.

Maybe something as:

```ts
async componentDidUpdate(prevProps, prevState) {

    const { filters } = this.state;
    /**
      Is verifying that isProcessing is true and that it has changed its value,
      since in the shouldComponentUpdate it is making it re-execute several times
      each time a state changes, regardless of whether it is a change in isProcessing.
     */
    if (
    (JSON.stringify(filters) !== JSON.stringify(prevState.filters))
    || (
       prevProps.state.isProcessing !== this.props.state.isProcessing &&
       this.props.state.isProcessing &&
       this._isMounted
     )
    ) {
      await this.getItems();
    }
```

Anyways, we need to test the proposed solution if this change doesn't cause other problems.

Reply:
Applied 👍🏼